src/libponyc/pkg/ifdef.c

```diff
@@ -72,7 +72,7 @@ static void cond_normalise(ast_t** astp)
               NODE(TK_IFDEFFLAG, ID(OS_LINUX_NAME))
               NODE(TK_IFDEFFLAG, ID(OS_MACOSX_NAME))
               NODE(TK_NONE))
-            NODE(TK_IFDEFFLAG, ID(OS_FREEBSD_NAME))
+            NODE(TK_IFDEFFLAG, ID(OS_BSD_NAME))
```

Comment:
i believe this is correct. we want any BSD. and we don't need freebsd or dragonfly specifically, correct?